### ISSUE TITLE
DRawSqlService might silently drop columns with duplicate names

### ISSUE THREAD

**Issue by focdanisch** [ebean-orm/ebean#1801]
Hi Rob,

We use DRawSqlService.sqlRow(...) to wrap the results we get from a prepared statement (executed manually without using ebean). However, we realized that if the row from the ResultSet contains multiple columns with the same name (e.g., the result of a table join might produce this), the mentioned method silently drops duplicate columns and only stores the last column in the map.

I produced a small modification which prefixes the column label or name with the schema and table name if the map contains the name. I also wrote a small test which queries the method and checks the results. The pull request can be found here: https://github.com/ebean-orm/ebean/pull/1800

For consistency reasons, we could always prefix the label or columns with the schema and table name. I can change the code accordingly, if the general procedure is OK for you.

Best regards,
Manuel

**Comment by rbygrave** [comment]
> For consistency reasons, we could always prefix the label or columns with the schema and table name. I can change the code accordingly, if the general procedure is OK for you.

I'm good with that.

My only thought is if some JDBC driver/database combination returns a null schema name.  That is, there might be an argument for only appending schema name if it is not null.  I've just added a small adjustment to do that (for the cases where a jdbc driver returns null schema or table name).

I've merged the code and pushed that minor adjustment.

I think we are all good.
